modules/HapticGlove_module/include/GloveControlHelper.hpp

```diff
@@ -115,6 +122,12 @@ class HapticGlove::GloveControlHelper
      */
     bool setPalmVibrotactileFeedbackReference(const int& desiredValue);
 
+    /**
+     * stop the haptics feedback (force feedback, fingertip vibrotactile feedback) to the user
+     * @return  true/false in case of success/failure
+     */
+    bool stopHapticsFeedback();
```

Comment:
Nitpick. This does not seem grammatically correct. What about ``stopAllHapticFeedback``?

Reply:
Yes, makes more sense. Done!